src/main/kotlin/org/team5419/fault/hardware/ctre/BerkeliumFX.kt

```diff
@@ -0,0 +1,42 @@
+package org.team5419.fault.hardware.ctre
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX
+import org.team5419.fault.math.units.SIUnit
+import org.team5419.fault.math.units.SIKey
+import org.team5419.fault.math.units.Meter
+import org.team5419.fault.math.units.Second
+import org.team5419.fault.math.units.inMilliseconds
+import org.team5419.fault.math.units.Ampere
+import org.team5419.fault.math.units.inAmps
+import org.team5419.fault.math.units.derived.Radian
+import org.team5419.fault.math.units.native.NativeUnitModel
+
+typealias LinearBerkeleiumSRX = BerkeliumSRX<Meter>
+typealias AngularBerkeleiumSRX = BerkeliumSRX<Radian>
+
+class BerkeliumFX<T : SIKey>(
+    val talonFX: TalonSRX,
+    model: NativeUnitModel<T>
+) : CTREBerkeliumMotor<T>(talonFX, model) {
+
+    constructor(id: Int, model: NativeUnitModel<T>): this(TalonSRX(id), model)
+
+    init {
+        talonFX.configFactoryDefault(0)
```

Comment:
Maybe set the feedback device to the integrated encoder by default? is that a good idea @connorworley ?